fetch/api/basic/scheme-blob.sub.any.js

```diff
@@ -45,4 +45,78 @@ invalidRequestMethods.forEach(function(method) {
 checkKoUrl("blob:not-backed-by-a-blob/", "GET",
            "Fetching [GET] blob:not-backed-by-a-blob/ is KO");
 
+let empty_blob = new Blob([]);
+checkFetchResponse(URL.createObjectURL(empty_blob), "", "", 0,
+                  "Fetching URL.createObjectURL(empty_blob) is OK");
+
+let empty_type_blob = new Blob([], {type: ""});
+checkFetchResponse(URL.createObjectURL(empty_type_blob), "", "", 0,
+                  "Fetching URL.createObjectURL(empty_type_blob) is OK");
+
+let empty_data_blob = new Blob([], {type: "text/plain"});
+checkFetchResponse(URL.createObjectURL(empty_data_blob), "", "text/plain", 0,
+                  "Fetching URL.createObjectURL(empty_data_blob) is OK");
+
+promise_test(function(test) {
+  return fetch("/images/blue.png").then(function(resp) {
+    return resp.blob();
+  }).then(function(image_blob) {
+    return image_blob.arrayBuffer();
```

Comment:
Wouldn't this suffice:
```suggestion
    return resp.arrayBuffer();
```
?

(The alternative would be to peek at the image data and copy the first couple of bytes, but this seems reasonable.)

Reply:
Neat! Yeah this totally works. For some reason I missed `Response.arrayBuffer`